bin/common.py

```diff
@@ -37,7 +37,7 @@
 class version(T.SimpleNamespace):
     # Executable versioning
     vault = "1.0.0"
-    sandman = "1.0.0"
+    sandman = "1.1.0"
```

Comment:
Under a system of merging feature branches to develop, then doing releases to master, as we're doing now for vault, you don't increment the version in a feature branch.

But since it's just one feature in the pipeline and I'm immediately going to release to master, no need to undo this on this occasion.